### ISSUE TITLE
Brexit.

### ISSUE THREAD

**Issue by Datavoore** [kdeldycke/vat-rates#17]
Hello everyone,

As you might have heard, UK has left the EU this January first. 

The last line from the csv (for UK) is : 

2011-01-04 |   | GB UK IM | GBP | 0.2 | standard | United Kingdom (member state) standard VAT rate. Isle of Man (British self-governing dependency) is member of the EU VAT area and subjected to UK's standard VAT rate.
-- | -- | -- | -- | -- | -- | --

Should a line be added with the 0 vat rate ? 

Thanks in advance,

**Comment by timwhitlock** [comment]
Not my repo, but my understanding is that it is not a EU specific database. Users must decide which rates apply to them, but the rates are the rates. For example, Switzerland is in there, but I don't have to charge it because I'm in the UK.

The UK standard VAT rate is still 20%. It's just the wording in the comment that needs changing. Maybe leaving a historical entry with EU wording, and a new one with UK wording.

**Comment by Datavoore** [comment]
You're right thanks, I was a bit confused there. Let's see what the owner wants to do with your suggestion of changing the comment.
